src/connector/src/common.rs

```diff
@@ -440,6 +441,23 @@ impl ClickHouseCommon {
     }
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone)]
```

Comment:
If this config is only used in redis, we should put in in the file of redis sink. ditto for clickhouse sink.